### PR TITLE
Fix image url

### PR DESCRIPTION
**Pull request by drososanastasios**
This url is used when we click the button to download an image.

### CODE REVIEW

File: Resources/public/js/FileUploader.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function PunkAveFileUploader(options)
       appendEditableImage({
         // cmsMediaUrl is a global variable set by the underscoreTemplates partial of MediaItems.html.twig
         'thumbnail_url': viewUrl + '/thumbnails/' + encodeURIComponent(file),
-        'url': viewUrl + '/originals/' + file,
+        'url': viewUrl + '/originals/' + encodeURIComponent(file),
         'name': file
```
Comment (llissssss) [inline Resources/public/js/FileUploader.js]: this file does not need encode¿

File: Resources/public/js/FileUploader.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ function PunkAveFileUploader(options)
       appendEditableImage({
         // cmsMediaUrl is a global variable set by the underscoreTemplates partial of MediaItems.html.twig
         'thumbnail_url': viewUrl + '/thumbnails/' + encodeURIComponent(file),
-        'url': viewUrl + '/originals/' + file,
+        'url': viewUrl + '/originals/' + encodeURIComponent(file),
```
Comment (llissssss) [inline Resources/public/js/FileUploader.js]: we could have variable with the encapsulated file as it's used in lines 23 and 24

File: Resources/public/js/FileUploader.js (hunk 1 above)
Comment (drososanastasios) [inline Resources/public/js/FileUploader.js]: No this name is used as a fileId so the value should not be changed. You could check for the usages of this name is file `fileupload-templates.html.twig`

File: Resources/public/js/FileUploader.js (hunk 2 above)
Comment (drososanastasios) [inline Resources/public/js/FileUploader.js]: Ok i will do that

**Pull request merged**